source/includes/code-examples/FindMany.cs

```diff
@@ -0,0 +1,96 @@
+using MongoDB.Bson;
+using MongoDB.Bson.Serialization.Attributes;
+using MongoDB.Bson.Serialization.Conventions;
+using MongoDB.Driver;
+using static System.Console;
```

Comment:
is it really needed? Sounds like you use different formats with `Console.WriteLine` in different places. I would recommend using the most standard: `Console.Writeline` (not just WriteLine + line 5)

Reply:
This isn't a functional issue and affects multiple pages in the docs, so I'll create a separate ticket to be addressed after the docs are released.

Reply:
The key point as I mentioned before is that you use 2 different formats (which way better is a different question). See: https://github.com/mongodb/docs-csharp/pull/51/files#diff-790e8dbf2076f80693f7b76b1e5c2603abce672af63c9d44f000e6ccb1ba1fc7R24 and https://github.com/mongodb/docs-csharp/pull/51/files#diff-57fa48a1757ed5fb36ac3003227f186eae73dd0fba0ef4b36c5c56999e8ae7f5R19. In this particular file, you don't need this configuration at all.

Reply:
Updated all `WriteLine` calls to `Console.WriteLine` and removed `using static System.Console`